src/document/docUtil.ts

```diff
@@ -29,7 +29,8 @@ export function getPageNumber(element: Element | null): number | undefined {
 
 export function getRange(): Range | null {
     const selection = window.getSelection();
-    if (!selection || selection.type !== 'Range' || selection.isCollapsed || !selection.rangeCount) {
+
+    if (!selection || selection.isCollapsed || !selection.rangeCount) {
```

Comment:
Should this be `selection.rangeCount > 1`, then?

Reply:
`isCollapsed === true` for `Caret`, also for normal range selection, the range count is also 1.